app/controllers/characters_controller.rb

```diff
@@ -30,6 +30,7 @@ def new
 
   def show
     @character = Character.find_by(slug: params[:id])
+    @bank_account = BankAccount.where('character_id =?', @character.id)
```

Comment:
Prefer `@bank_account = BankAccount.where(character: @character)`.

Reply:
Also, do we really need to get this separately? Can we not just lazy load by calling `@character.bank_account`, since it's a `has_one` relationship?

Reply:
refactored using lazy load in the HAML template.